package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "ecies-geth",
-  "version": "1.7.4",
+  "version": "1.7.5",
   "description": "JavaScript Elliptic Curve Integrated Encryption Scheme (ECIES) Library - Based off Geth's implementation",
   "main": "dist/lib/src/typescript/index.js",
-  "types": "index.d.ts",
   "browser": "dist/lib/src/typescript/browser.js",
+  "types": "dist/lib/src/typescript/index.d.ts",
```

Comment:
Take a look at this file, it's empty, whereas it should be +/- the same contents as the root (deleted) index.d.ts